From 99a40ca87adb8c15cb9e454ae8e5a9c0a8e7172c Mon Sep 17 00:00:00 2001
From: Leon Anavi <leon.anavi@konsulko.com>
Date: Thu, 6 Nov 2025 17:07:04 +0000
Subject: [PATCH] mnistCUDNN/CMakeLists.txt: Fix FreeImage

Remove the private FreeImage and use the global freeimage.

Starting with CUDA 11.2, FreeImage is no longer distributed with
the CUDA Samples:
https://docs.nvidia.com/cuda/archive//11.5.0/cuda-samples/index.html#new-features-in-cuda-toolkit-11-2

Upstream-Status: Inappropriate [OE-Specific]

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
 usr/src/cudnn_samples_v9/mnistCUDNN/CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/usr/src/cudnn_samples_v9/mnistCUDNN/CMakeLists.txt b/usr/src/cudnn_samples_v9/mnistCUDNN/CMakeLists.txt
index 7fd0e09..1ead338 100644
--- a/usr/src/cudnn_samples_v9/mnistCUDNN/CMakeLists.txt
+++ b/usr/src/cudnn_samples_v9/mnistCUDNN/CMakeLists.txt
@@ -1,5 +1,3 @@
-add_subdirectory(FreeImage)
-
 add_executable(
     mnistCUDNN
     mnistCUDNN.cpp
@@ -12,7 +10,7 @@ add_test(
 )
 target_link_libraries(
     mnistCUDNN
-    PRIVATE FreeImage
+    freeimage
     CUDA::cublas
     CUDA::cudart_static
     cudnn
-- 
2.47.3

